content/kubermatic/master/_index.en.md

```diff
@@ -9,30 +9,73 @@ date = 2019-04-27T16:06:34+02:00
+
 
 KKP is directly integrated with leading cloud providers such as [Amazon Web Services](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/aws/aws/), [Azure](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/azure/azure/), DigitalOcean, [Google Compute Engine](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/google_cloud/gcp/), Hetzner, OpenStack, Packet and [VMware vSphere](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/vsphere/vsphere/) as well as any provider offering Ubuntu 16.04 or greater, even in your own datacenter.
 
-## Want to start with KKP in a GitOps way?
-Check the [Start with KKP]({{< ref "./installation/startio/" >}}) for more details.
 
 ## Features
 
-{{% notice tip %}}
-For latest updates follow us on Twitter [@Kubermatic](https://twitter.com/Kubermatic)
-{{% /notice %}}
+#### Powerful & intuitive dashboard to visualize your Kubernetes deployment
+Manage your projects and clusters with the KKP dashboard. Scale your cluster by adding and removing nodes in just a few clicks. As an Admin, the dashboard also allows you to customize the theme and disable theming options for other users.
+
+#### Deploy, scale & update multiple Kubernetes clusters
+Kubernetes environments must be highly distributed to meet the performance demands of modern cloud native applications. Organizations can ensure consistent operations across all environments with effective cluster management. KKP empowers you to take advantage of all the advanced features that Kubernetes has to offer and increases the speed, flexibility and scalability of your cloud deployment workflow. 
+
+At Kubermatic, we have chosen to do multi-cluster management with Kubernetes Operators. Operators (a method of packaging, deploying and managing a Kubernetes application) allow KKP to automate creation as well as the full lifecycle management of clusters. With KKP you can create a cluster for each need, fine-tune it, reuse it and continue this process hassle-free. This results in:
+- Better failure resilience
+- Smaller individual clusters being more adaptable than one big cluster
+- Faster development thanks to less complex environments
+
+#### Kubernetes Autoscaler
+Autoscaling in Kubernetes refers to the ability to increase or decrease the number of nodes as the demand for service response changes. Without autoscaling, teams would manually first provision and then scale up or down resources every time conditions change. This means, either services fail at peak demand due to the unavailability of enough resources or you pay at peak capacity to ensure availability. 
```

Comment:
> This means, either services fail at peak demand due to the unavailability of enough resources or you pay at peak capacity to ensure availability.

I believe you want to say that users need to pay for more resources than they need, but it doesn't read that way to me. This part is particularly confusing:

> ... or you pay at peak capacity to ensure availability.

I guess it might be better to say something along (note `all the time`):

> ... or you pay for resources at the peak capacity all the time to ensure availability.

Reply:
I would keep it this way

Reply:
Okay, let's keep it this way.